tshirts.cpp

```diff
@@ -15,7 +15,11 @@ char size(int cms) {
 
 int main() {
     assert(size(37) == 'S');
+	assert(size(38) == 'S');

```

Comment:
One of these asserts will fail, irrespective of how you fix the code. Make up your mind on the size of `38` and code one assert for it.